dspace-api/src/main/java/org/dspace/submit/step/AccessStep.java

```diff
@@ -7,318 +7,28 @@
-
-    public static int checkForm(HttpServletRequest request){
-
-        String selectedRadio=null;
-        String dateEmbargoUntil = request.getParameter("embargo_until_date");
-
-
-        // RADIO_OPEN_ACCESS_ITEM_VISIBLE=0;
-        // RADIO_OPEN_ACCESS_ITEM_EMBARGOED=1;
-        if((selectedRadio=request.getParameter("open_access_radios"))!=null && Integer.parseInt(selectedRadio)==1
-                && (dateEmbargoUntil==null || dateEmbargoUntil.equals(""))){
-            return STATUS_ERROR_MISSING_DATE;
-        }
-
-
-        if(dateEmbargoUntil !=null && !dateEmbargoUntil.equals("")){
-            Date startDate = getEmbargoUntilDate(request);
-            if(startDate==null){
-                return STATUS_ERROR_FORMAT_DATE;
-            }
-        }
-        return 0;
-    }
-
-
-
-
-
-
-
-    public static Date getEmbargoUntil(HttpServletRequest request) {
-        // RADIO_OPEN_ACCESS_ITEM_VISIBLE=0;
-        // RADIO_OPEN_ACCESS_ITEM_EMBARGOED=1;
-        String selectedRadio;
-        Date dateStartDate=null;
-        if((selectedRadio=request.getParameter("open_access_radios"))!=null && Integer.parseInt(selectedRadio)==1){
-            Date startDate = getEmbargoUntilDate(request);
-            if(startDate!=null) dateStartDate=startDate;
-        }
-        return dateStartDate;
-    }
-
-    private static Date getEmbargoUntilDate(HttpServletRequest request) {
-        Date startDate = null;
-        try {
-            startDate = DateUtils.parseDate(request.getParameter("embargo_until_date"), new String[]{"yyyy-MM-dd", "yyyy-MM", "yyyy"});
-        } catch (Exception e) {
-            //Ignore start date is already null
-        }
-        return startDate;
-    }
-
-    /**
-     * Retrieves the number of pages that this "step" extends over. This method
-     * is used to build the progress bar.
-     * <P>
-     * This method may just return 1 for most steps (since most steps consist of
-     * a single page). But, it should return a number greater than 1 for any
-     * "step" which spans across a number of HTML pages. For example, the
-     * configurable "Describe" step (configured using submission-forms.xml) overrides
-     * this method to return the number of pages that are defined by its
-     * configuration file.
-     * <P>
-     * Steps which are non-interactive (i.e. they do not display an interface to
-     * the user) should return a value of 1, so that they are only processed
-     * once!
-     * 
-     * @param request
-     *            The HTTP Request
-     * @param subInfo
-     *            The current submission information object
-     * 
-     * @return the number of pages in this step
-     */
-    @Override
-    public int getNumberOfPages(HttpServletRequest request,
-            SubmissionInfo subInfo) throws ServletException
-    {
-        return 1;
-
-    }
+	@Override
+	public void doProcessing(Context context, Request req, InProgressSubmission wsi) {
+		// TODO Auto-generated method stub
```

Comment:
I'd like to see a more informative comment here. There's several areas of this PR which have this exact same `// TODO Auto-generated method stub` comment.  If these methods should be empty, let's add in a comment that says why they are empty, instead of having a `TODO` statement.

Reply:
Also, it is unclear to me why several of these classes are being kept with entirely empty methods.  Are these classes (such as this `AccessStep` one) now deprecated?  (If so, they should be flagged as such)  Is there some other reason they are not just deleted?